docs/areatrigger_scripts.md

```diff
@@ -10,8 +10,8 @@ Allows for an area trigger to be scripted with Trinity Script.
 
 | Field           | Type         | Attributes | Key | Null | Default | Extra | Comment |
 |-----------------|--------------|------------|-----|------|---------|-------|---------|
-| [entry][1]      | mediumint(8) | signed     | PRI | NO   | NULL    |       |         |
-| [ScriptName][2] | char(64)     | signed     |     | NO   | NULL    |       |         |
+| [entry][1]      | mediumint(8) |            | PRI | NO   |         |       |         |
+| [ScriptName][2] | char(64)     |            |     | NO   |         |       |         |
```

Comment:
NULL value  as default is a good info

Reply:
I don't think I'm getting this right. According to the SQL script, the fields are not null.

```sql
CREATE TABLE `areatrigger_scripts` (
  `entry` mediumint(8) NOT NULL,
  `ScriptName` char(64) NOT NULL,
  PRIMARY KEY (`entry`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8
```

But maybe we are talking about different subjects and I am misunderstanding.
Anything, let me know, to make the arrangement.

Reply:
it seems that NOT NULL does not mean the default value
![image](https://user-images.githubusercontent.com/519778/104600567-73470d80-5679-11eb-86a6-3ff8e7bb2231.png)

Keep as you wish ;-)

Reply:
I get a lot of confusion at first about that subject. Until more or less I think I end up holding his hand, anyway, just in case, when I have doubts, I check the SQL. Thanks for the feedback.